get.sh

```diff
@@ -160,6 +160,34 @@ parseCommandLineArgs()
 	echo "TESTDIR: $TESTDIR"
 }
 
+# If the current directory contains only a single directory then squash that directory into the current directory
+squashSingleFolderContentsToCurrentDir()
+{
+        # Does current directory contain one and ONLY one directory?
+        if [[ `ls -d */ | wc -l` -eq 1 ]] && [[ `ls -d * | wc -l` -eq 1 ]]; then
```

Comment:
Alternate option which seems much more readable IMHO
```suggestion
        if [[ $(ls -1 | wc -l) -eq 1 ]]; then
          folder=$(ls -d */)
            if [ -d "$folder" ]; then
```
Noting that I'm suggesting $(...) instead of backticks because the linter typically objects to those - I'm surprised it hasn't objected in this case as it's picked me up on it quite a lot (although I personally prefer the backticks!)

Reply:
I used them to be consistent with what's used in this method already, but yeah usually $()

Reply:
done